scraper.rb

```diff
@@ -1,14 +1,21 @@
 #!/bin/env ruby
 # encoding: utf-8
 require 'scraperwiki'
-require 'colorize'
-require 'combine_popolo_memberships'
+require 'require_all'
+require 'scraped_page_archive'
 
-require_relative 'lib/members.rb'
+require_rel 'lib'
 
 members_url = 'http://api.openhluttaw.org/en/memberships'
-members = Members.new(members_url).to_h[:members_of_the_lower_house]
 
-members.each do |member|
-  ScraperWiki.save_sqlite([:id], member.to_h)
+def scrape_all_members(url)
```

Comment:
This is confusingly named. In conjunction with similarly named 'current_members", and "members_of_the_lower_house", the implication is that this is dealing with _all_ members. And thus it's confusing when it calls itself repeatedly. It would be good to make it more obvious that this is about pagination.